client/src/auth/slice.js

```diff
@@ -58,6 +59,22 @@ const authSlice = createSlice({
         return { payload: { history } };
       },
     },
+    refreshActiveGames: {
+      reducer: (state, action) => {
+        if (state.currentUser.id !== action.payload.id) {
+          // what should we do when refreshing with an ID we do not expect?
+          throw `bad user id: expected "${state.currentUser.id}", got "${action.payload.id}"`;
+        }
+      },
+      prepare: (playerID) => {
+        return { payload: { id: playerID } };
+      },
```

Comment:
Do we need a custom `prepare` function here, or can we let RTK just assign `playerID` to the payload in its default way?

Reply:
tbh I have no idea what should be done here; I'll try pulling it out